### PR TITLE
more German translations for complete Map

### PR DESCRIPTION
**Pull request by ReuschelCGN**
1. more/complete german translation for RocketMAD (Mons, Items, Menues, Functions etc...) File ist completely sort alphabetical.
2. fix steel rocket grunts for menu and update Govanni Mons for April

### CODE REVIEW

File: static/locales/de.json [hunk 1]
```diff
@@ -543,61 +570,87 @@
...
   "Gulpin": "Schluppuck",
+  "Gumshoos": "Manguspektor",
   "Gunk Shot": "Mülltreffer",
   "Gurdurr": "Strepoli",
   "Gust": "Windstoß",
+  "Guzzlord": "Schlingking",
   "Gyarados": "Garados",
+  "Gym cells": "Arena Zellen",
+  "Gym leader": "Arenaboss",
+  "Gym levels": "Raidlevel",
```
Comment (pki0) [inline static/locales/de.json:638]: That is not the raid level. It's the gym level.

File: static/locales/de.json [hunk 2]
```diff
@@ -543,61 +570,87 @@
...
+  "Gym sidebar": "Arena-Seitenleiste",
+  "Gyms & Raids": "Arenen & Raids",
+  "Gyms in battle only": "nur Arenen mit Raidkampf",
```
Comment (pki0) [inline static/locales/de.json:641]: Capital

File: static/locales/de.json [hunk 3]
```diff
@@ -617,26 +670,29 @@
...
+  "Height": "Größe",
   "Heliolisk": "Elezard",
   "Helioptile": "Eguana",
   "Helping Hand": "Rechte Hand",
   "Heracross": "Skaraborn",
   "Herdier": "Terribark",
   "Hex": "Bürde",
   "Hidden Power": "Kraftreserve",
+  "Hide Pokémon": "Pokémon ausblenden",
+  "Hide": "ausblenden",
```
Comment (pki0) [inline static/locales/de.json:682]: Capital

File: static/locales/de.json [hunk 4]
```diff
@@ -617,26 +670,29 @@
...
   "Hitmonchan": "Nockchan",
   "Hitmonlee": "Kicklee",
   "Hitmontop": "Kapoera",
-  "Ho-Oh": "Ho-Oh",
   "Hold Back": "Zurückhaltung",
   "Hold Hands": "Händchenhalten",
   "Honchkrow": "Kramshef",
   "Hone Claws": "Klauenwetzer",
   "Honedge": "Gramokles",
+  "Ho-Oh": "Ho-Oh",
```
Comment (pki0) [inline static/locales/de.json:695]: Not needed

File: static/locales/de.json [hunk 5]
```diff
@@ -701,36 +768,56 @@
...
-  "Land's Wrath": "Bodengewalt",
   "Landorus": "Demeteros",
+  "Land's Wrath": "Bodengewalt",
   "Lanturn": "Lanturn",
   "Lapras": "Lapras",
+  "Large" : "Groß",
   "Larvesta": "Ignivor",
   "Larvitar": "Larvitar",
   "Laser Focus": "Konzentration",
+  "Last 12 hours": "den letzten 12 Stunden",
```
Comment (pki0) [inline static/locales/de.json:806]: sugg -> "Letzte 12 Stunden". Capital.

File: static/locales/de.json [hunk 6]
```diff
@@ -701,36 +768,56 @@
...
+  "Last 14 days": "den letzten 14 Tagen",
+  "Last 3 hours": "den letzten 3 Stunden",
+  "Last 3 month": "den letzten 3 Monaten",
+  "Last 6 hours": "den letzten 6 Stunden",
+  "Last 6 month": "den letzten 6 Monaten",
+  "Last 7 days": "den letzten 7 Tagen",
+  "Last day": "den letzten 24 Stunden",
+  "Last hour": "der letzten Stunde",
+  "Last modified": "zuletzt geändert",
+  "Last month": "den letzten 30 Tagen",
```
Comment (pki0) [inline static/locales/de.json:816]: s.a. Capital

File: static/locales/de.json [hunk 7]
```diff
@@ -701,36 +768,56 @@
...
   "Last Resort": "Zuflucht",
+  "Last scanned": "zuletzt gescannt",
+  "Last seen": "zuletzt gesehen",
+  "Last year": "den letzten 12 Monaten",
```
Comment (pki0) [inline static/locales/de.json:820]: Capital. "Last year" = "Letztes Jahr"

File: static/locales/de.json [hunk 8]
```diff
@@ -795,26 +893,38 @@
   "Magnet Bomb": "Magnetbombe",
   "Magnet Rise": "Magnetflug",
   "Magnetic Flux": "Magnetregler",
+  "Magnetic Lure": "Magnet-Lockmodul",
+  "Magnetic Lures": "Magnet-Lockmodul",
   "Magneton": "Magneton",
   "Magnezone": "Magnezone",
   "Magnitude": "Intensität",
+  "Main weather in header": "Wetter im Kopf anzeigen",
```
Comment (pki0) [inline static/locales/de.json:901]: sugg -> Wetter oben anzeigen.

File: static/locales/de.json [hunk 9]
```diff
@@ -795,26 +893,38 @@
...
   "Makuhita": "Makuhita",
   "Malamar": "Calamanero",
+  "Male": "männlich",
```
Comment (pki0) [inline static/locales/de.json:904]: Capital

File: static/locales/de.json [hunk 10]
```diff
@@ -795,26 +893,38 @@
...
   "Mamoswine": "Mamutel",
   "Manaphy": "Manaphy",
   "Mandibuzz": "Grypheldis",
   "Manectric": "Voltenso",
   "Mankey": "Menki",
   "Mantine": "Mantax",
   "Mantyke": "Mantirps",
+  "Map lifetime": "seit Start der Karte",
```
Comment (pki0) [inline static/locales/de.json:912]: Capital

File: static/locales/de.json [hunk 11]
```diff
@@ -795,26 +893,38 @@
...
+  "Map misc": "Karte weiteres",
+  "Map style": "Map Style",
```
Comment (pki0) [inline static/locales/de.json:914]: Here no translation of "map"  but above and after?

File: static/locales/de.json [hunk 12]
```diff
@@ -825,16 +935,17 @@
   "Mega Punch": "Megahieb",
   "Megahorn": "Vielender",
   "Meganium": "Meganie",
+  "Melmetal": "Melmetal",
```
Comment (pki0) [inline static/locales/de.json:938]: Not needed

File: static/locales/de.json [hunk 13]
```diff
@@ -868,30 +982,46 @@
...
   "Mystical Fire": "Magieflamme",
+  "Naganadel": "Agoyon",
+  "Name includes": "Name beinhaltet",
+  "Nanab Berry": "Nanabbeere(n)",
   "Nasty Plot": "Ränkeschmied",
   "Natu": "Natu",
   "Natural Gift": "Beerenkräfte",
   "Nature Power": "Natur-Kraft",
   "Nature's Madness": "Naturzorn",
+  "Necrozma": "Necrozma",
```
Comment (pki0) [inline static/locales/de.json:1021]: Not needed.

File: static/locales/de.json [hunk 14]
```diff
@@ -902,19 +1032,35 @@
...
+  "Normal Lure": "Lockmodul",
+  "Normal Lures": "Lockmodul",
   "Normal": "Normal",
   "Nosepass": "Nasgnet",
+  "Notif IVs": "Alarm IV",
+  "Notif Levels": "Alarm Level",
+  "Notif Pokémon filtered by stats": "Pokémonalarm mit Statuswertfilter",
+  "Notif Raid Bosses": "Raid Boss Alarm",
+  "Notif Rocket Invasions": "Rocket Invasion Alarm",
+  "Notif Quest Pokémon": "Feldforschungsalarm Pokémon",
```
Comment (pki0) [inline static/locales/de.json:1053]: Somewhere above "Quest" was not translated. Would not do this.

File: static/locales/de.json [hunk 15]
```diff
@@ -923,13 +1069,19 @@
   "Omastar": "Amoroso",
   "Ominous Wind": "Unheilböen",
   "Onix": "Onix",
+  "Open spot only": "nur mit offenen Plätzen",
```
Comment (pki0) [inline static/locales/de.json:1072]: Capital

File: static/locales/de.json [hunk 16]
```diff
@@ -923,13 +1069,19 @@
   "Omastar": "Amoroso",
   "Ominous Wind": "Unheilböen",
   "Onix": "Onix",
```
Comment (pki0) [inline static/locales/de.json:1071]: Not needed.

File: static/locales/de.json [hunk 17]
```diff
@@ -962,32 +1118,49 @@
...
   "Piloswine": "Keifel",
   "Pin Missile": "Nadelrakete",
+  "Pinap Berry": "Sananabeere(n)",
+  "Pincurchin": "Britzigel",
   "Pineco": "Tannza",
   "Pinsir": "Pinsir",
   "Piplup": "Plinfa",
   "Play Nice": "Kameradschaft",
   "Play Rough": "Knuddler",
+  "Play sound on notif": "spiele Alarmton",
```
Comment (pki0) [inline static/locales/de.json:1131]: Capital

File: static/locales/de.json [hunk 18]
```diff
@@ -962,32 +1118,49 @@
...
   "Poison Gas": "Giftwolke",
   "Poison Jab": "Gifthieb",
   "Poison Powder": "Giftpuder",
   "Poison Sting": "Giftstachel",
   "Poison Tail": "Giftschweif",
   "Poison": "Gift",
+  "Poké Ball": "Pokéball",
+  "Pokémon filtered by stats": "Pokémon Statuswertfilter",
+  "Pokémon notifs": "Pokémon Alarm",
+  "Pokémon seen in": "gesehene Pokémon in",
```
Comment (pki0) [inline static/locales/de.json:1145]: Capital

File: static/locales/de.json [hunk 19]
```diff
@@ -962,32 +1118,49 @@
...
   "Poliwrath": "Quappo",
   "Pollen Puff": "Pollenknödel",
+  "Polteageist": "Mortipot",
   "Ponyta": "Ponita",
   "Poochyena": "Fiffyen",
+  "Popplio": "Robball",
   "Porygon": "Porygon",
-  "Porygon-Z": "Porygon-Z",
   "Porygon2": "Porygon2",
+  "Porygon-Z": "Porygon-Z",
```
Comment (pki0) [inline static/locales/de.json:1162]: Not needed.

File: static/locales/de.json [hunk 20]
```diff
@@ -1026,36 +1200,54 @@
   "Pursuit": "Verfolgung",
   "Purugly": "Shnurgarst",
   "Pyroar": "Pyroleo",
+  "Pyukumuku": "Gufa",
   "Quagsire": "Morlord",
   "Quash": "Verzögerung",
+  "QUEST ITEMS": "Feldforschungsitems",
+  "Quest notifs": "Feldforschungsalarm",
+  "QUEST POKÉMON": "Feldforschungspokémon",
```
Comment (pki0) [inline static/locales/de.json:1208]: Sugg -> No translate of quest.

File: static/locales/de.json [hunk 21]
```diff
@@ -1026,36 +1200,54 @@
...
+  "Razz Berry": "Himmihbeere(n)",
   "Recover": "Genesung",
   "Recycle": "Aufbereitung",
+  "Red-Striped": "Rotlinig",
   "Reflect Type": "Typenspiegel",
   "Reflect": "Reflektor",
   "Refresh": "Heilung",
   "Regice": "Regice",
+  "Regidrago": "Regidrago",
+  "Regieleki": "Regieleki",
```
Comment (pki0) [inline static/locales/de.json:1250]: Not needed.

File: static/locales/de.json [hunk 22]
```diff
@@ -1085,37 +1282,53 @@
...
   "Sandshrew": "Sandan",
   "Sandslash": "Sandamer",
   "Sandstorm": "Sandsturm",
+  "Sandygast": "Sankabuh",
+  "satellite": "Satellit",
   "Sawk": "Karadonis",
   "Sawsbuck": "Kronjuwild",
   "Scald": "Siedewasser",
+  "Scanned locations": "gescannte Orte",
+  "Scanned": "gescannt",
```
Comment (pki0) [inline static/locales/de.json:1325]: Capital

File: static/locales/de.json [hunk 23]
```diff
@@ -1131,6 +1344,8 @@
   "Seed Flare": "Schocksamen",
   "Seedot": "Samurzel",
   "Seel": "Jurob",
+  "Seen (%)": "gesehen (%)",
+  "Seen": "gesehen",
```
Comment (pki0) [inline static/locales/de.json:1348]: Capital

File: static/locales/de.json [hunk 24]
```diff
@@ -1159,21 +1374,31 @@
   "Shieldon": "Schilterus",
   "Shift Gear": "Gangwechsel",
   "Shiftry": "Tengulist",
+  "Shiinotic": "Lamellux",
   "Shinx": "Sheinux",
   "Shock Wave": "Schockwelle",
   "Shore Up": "Sandsammler",
+  "Show notif Pokémon only": "zeige nur gewählte Pokémon",
```
Comment (pki0) [inline static/locales/de.json:1381]: Capital

File: static/locales/de.json [hunk 25]
```diff
@@ -1275,19 +1518,28 @@
   "String Shot": "Fadenschuss",
   "Struggle Bug": "Käfertrutz",
   "Struggle": "Verzweifler",
+  "Stufful": "Velursi",
   "Stun Spore": "Stachelspore",
   "Stunfisk": "Flunschlik",
   "Stunky": "Skunkapuh",
+  "style_light2": "Licht2",
+  "style_light2_nl": "Licht2 (Keine Labels)",
```
Comment (pki0) [inline static/locales/de.json:1526]: Sugg -> "Hell" not "Licht"

File: static/locales/de.json [hunk 26]
```diff
@@ -1275,19 +1518,28 @@
...
+  "style_pgo": "RocketMap",
+  "style_pgo_nl": "RocketMap (Keine Labels)",
   "Submission": "Überroller",
   "Substitute": "Delegator",
   "Sucker Punch": "Tiefschlag",
   "Sudowoodo": "Mogelbaum",
   "Suicune": "Suicune",
+  "Summer": "Sommerform",
```
Comment (pki0) [inline static/locales/de.json:1534]: Summer = Sommer. Would adjust somewhere else.

File: static/locales/de.json [hunk 27]
```diff
@@ -1346,9 +1608,13 @@
   "Thunder": "Donner",
   "Thunderbolt": "Donnerblitz",
   "Thundurus": "Voltolos",
+  "Thwackey": "Chimstix",
   "Tickle": "Spaßkanone",
   "Timburr": "Praktibalk",
+  "Tiny Rattata notifs": "winzige Rattfratz Alarm",
```
Comment (pki0) [inline static/locales/de.json:1614]: Capital

File: static/locales/de.json [hunk 28]
```diff
@@ -1357,12 +1623,17 @@
   "Torkoal": "Qurtel",
   "Torment": "Folterknecht",
   "Tornadus": "Boreos",
+  "Torracat": "Miezunder",
   "Torterra": "Chelterrar",
   "Totodile": "Karnimani",
+  "Toucannon": "Tukanon",
+  "Toxapex": "Aggrostella",
+  "Toxel": "Toxel",
```
Comment (pki0) [inline static/locales/de.json:1631]: Not needed

File: static/locales/de.json [hunk 29]
```diff
@@ -1376,32 +1647,42 @@
...
+  "Trumbeak": "Trompeck",
   "Trump Card": "Trumpfkarte",
+  "Tsareena": "Fruyal",
+  "Turtonator": "Tortunator",
   "Turtwig": "Chelast",
   "Twineedle": "Duonadel",
   "Twister": "Windhose",
   "Tympole": "Schallquap",
   "Tynamo": "Zapplardin",
+  "Type: Null": "Typ:Null",
```
Comment (pki0) [inline static/locales/de.json:1659]: Not needed

File: static/locales/de.json [hunk 30]
```diff
@@ -1376,32 +1647,42 @@
...
+  "Ultra Ball": "Hyperball",
   "Ultra Rare": "Ultra Selten",
   "Umbreon": "Nachtara",
   "Uncommon": "Nicht Häufig",
+  "Uncontested": "Teamlos",
   "Unfezant": "Fasasnob",
+  "Unova Stone" : "Einall-Stein",
   "Unown": "Icognito",
   "Uproar": "Aufruhr",
+  "Upscale notif markers": "vergrößer Alarmmarker",
```
Comment (pki0) [inline static/locales/de.json:1674]: Capital

File: static/locales/de.json [hunk 31]
```diff
@@ -1432,6 +1714,7 @@
   "Wailord": "Wailord",
   "Wake-Up Slap": "Weckruf",
   "Walrein": "Walraisa",
+  "Warn about hidden cells": "bei versteckten Zellen warnen",
```
Comment (pki0) [inline static/locales/de.json:1717]: Capital

File: static/locales/de.json [hunk 32]
```diff
@@ -1457,14 +1744,27 @@
   "Wigglytuff": "Knuddeluff",
   "Wild Charge": "Stromstoß",
   "Will-O-Wisp": "Irrlicht",
+  "Wimpod": "Reißlaus",
+  "Windy": "Windig",
   "Wing Attack": "Flügelschlag",
   "Wingull": "Wingull",
+  "Winter": "Winterform",
```
Comment (pki0) [inline static/locales/de.json:1751]: Wrong translation of "Winter"

File: static/locales/de.json [hunk 33]
```diff
@@ -1475,20 +1775,30 @@
...
   "Yamask": "Makabaja",
+  "Yamper": "Voldi",
   "Yanma": "Yanma",
   "Yanmega": "Yanmega",
   "Yawn": "Gähner",
+  "Yesterday": "Gestern",
+  "Yungoos": "Mangunior",
   "Yveltal": "Yveltal",
+  "Zacian": "Zacian",
+  "Zamazenta": "Zamazenta",
```
Comment (pki0) [inline static/locales/de.json:1793]: Not needed.

**Review by pki0 — CHANGES_REQUESTED** [review]
Made some suggestions. May take a look at #207 .

File: static/locales/de.json (hunk 33 above)
Comment (ReuschelCGN) [reply static/locales/de.json]: delete

File: static/locales/de.json (hunk 32 above)
Comment (ReuschelCGN) [reply static/locales/de.json:1749]: https://pokefans.net/spiele/pokemon-go/pokemon/sesokitz-winterform
see autum reply

File: static/locales/de.json (hunk 31 above)
Comment (ReuschelCGN) [reply static/locales/de.json]: fixed

File: static/locales/de.json (hunk 30 above)
Comment (ReuschelCGN) [reply static/locales/de.json]: fixed

File: static/locales/de.json (hunk 29 above)
Comment (ReuschelCGN) [reply static/locales/de.json]: delete (was in original file)

File: static/locales/de.json [hunk 34]
```diff
@@ -166,41 +105,53 @@
   "Belly Drum": "Bauchtrommel",
   "Bergmite": "Arktip",
   "Bestow": "Offerte",
+  "Bewear": "Kosturso",
   "Bibarel": "Bidifas",
   "Bide": "Geduld",
   "Bidoof": "Bidiza",
+  "Big Magikarp notifs": "große Karpador Alarm",
```
[the remark this answers is not in the document]
Comment (ReuschelCGN) [reply static/locales/de.json]: fixed

File: static/locales/de.json [hunk 35]
```diff
@@ -107,33 +37,41 @@
...
-  "Attack Order": "Schlagbefehl",
+  "Attack Order": "Angriffsreihenfolge",
   "Attract": "Anziehung",
   "Audino": "Ohrdoch",
   "Aura Sphere": "Aurasphäre",
   "Aurora Beam": "Aurorastrahl",
   "Aurora Veil": "Auroraschleier",
   "Aurorus": "Amagarga",
   "Autotomize": "Autotomie",
+  "Autumn": "Herbstform",
```
[the remark this answers is not in the document]
Comment (ReuschelCGN) [reply static/locales/de.json:74]: https://pokefans.net/spiele/pokemon-go/pokemon/sesokitz-herbstform 
Name in german ist Sesokitz Herbstform!

File: static/locales/de.json [hunk 36]
```diff
@@ -96,9 +23,12 @@
   "Air Cutter": "Windschnitt",
   "Air Slash": "Luftschnitt",
   "Alakazam": "Simsala",
+  "Alcremie": "Pokusan",
+  "All locations": "alle Gebiete",
   "Ally Switch": "Seitentausch",
   "Alomomola": "Mamolida",
   "Altaria": "Altaria",
+  "Always show notif Pokémon": "zeige immer gewählte Pokémon",
```
[the remark this answers is not in the document]
Comment (ReuschelCGN) [reply static/locales/de.json]: fixed

File: static/locales/de.json [hunk 37]
```diff
@@ -96,9 +23,12 @@
   "Air Cutter": "Windschnitt",
   "Air Slash": "Luftschnitt",
   "Alakazam": "Simsala",
+  "Alcremie": "Pokusan",
+  "All locations": "alle Gebiete",
```
[the remark this answers is not in the document]
Comment (ReuschelCGN) [reply static/locales/de.json]: fixed (but its no sentence, so capital letter is wrong by german grammar!)

File: static/locales/de.json [hunk 38]
```diff
@@ -218,26 +169,35 @@
...
   "Camouflage": "Tarnung",
   "Captivate": "Liebreiz",
   "Carbink": "Rocara",
+  "Carkol": "Wagong",
   "Carnivine": "Venuflibis",
   "Carracosta": "Karippas",
   "Carvanha": "Kanivanha",
   "Cascoon": "Panekon",
   "Castform": "Formeo",
+  "Catch {0} pokémon with berrie(s)": "Fange {0} Pokémon mit Beeren",
```
[the remark this answers is not in the document]
Comment (ReuschelCGN) [reply static/locales/de.json]: not from me, but delete from file

**Review by ReuschelCGN — COMMENTED** [review]
changes made

**Comment by pki0** [comment]
> changes made

I think you missed the 38 collapsed ones. :)

**Comment by pki0** [comment]
Did another quick look.
You translated some moves to wrong names. Like "charge" (is a move) or "attack order" (also a move). Please be careful with these. Maybe there are more.

File: static/locales/de.json [hunk 39]
```diff
@@ -218,26 +169,35 @@
...
   "Celebi": "Celebi",
   "Celebrate": "Ehrentag",
+  "Celesteela": "Kaguron",
+  "Centiskorch": "Infernopod",
   "Chandelure": "Skelabra",
   "Chansey": "Chaneira",
   "Charge Beam": "Ladestrahl",
-  "Charge": "Ladevorgang",
+  "Charge move": "Ladeattacke",
+  "Charge": "Ladeattacke",
```
[the remark this answers is not in the document]
Comment (pki0) [reply static/locales/de.json:195]: I was wrong here. Charge is a move: See https://www.pokewiki.de/Ladevorgang

Don't know if its in PoGo implemented.

File: static/locales/de.json [hunk 40]
```diff
@@ -442,20 +453,30 @@
   "Fairy": "Fee",
   "Fake Out": "Mogelhieb",
   "Fake Tears": "Trugträne",
+  "Falinks": "Legios",
   "False Swipe": "Trugschlag",
   "Farfetch'd": "Porenta",
+  "Fast move": "Sofortattacke",
+  "Fast TM": "Sofort TM",
+  "Fast": "Sofortattacke",
```
[the remark this answers is not in the document]
Comment (pki0) [reply static/locales/de.json:458]: Does "Sofortattacke" fit in the label and doesn't blow it up a lot?

File: static/locales/de.json [hunk 41]
```diff
@@ -495,21 +515,26 @@
   "Focus Blast": "Fokusstoß",
   "Focus Energy": "Energiefokus",
   "Focus Punch": "Power-Punch",
+  "Fog": "Nebel",
   "Follow Me": "Spotlight",
+  "Follow me", "Folge mir",
```
Comment (pki0) [inline static/locales/de.json:520]: Error

File: static/locales/de.json [hunk 42]
```diff
@@ -543,61 +568,87 @@
...
   "Gurdurr": "Strepoli",
   "Gust": "Windstoß",
+  "Guzzlord": "Schlingking",
   "Gyarados": "Garados",
+  "Gym cells": "Arena Zellen",
+  "Gym leader": "Arenaboss",
+  "Gym levels": "Arenalevel",
+  "Gym sidebar": "Arena-Seitenleiste",
+  "Gyms & Raids": "Arenen & Raids",
+  "Gyms in battle only": "Nur Arenen mit Raidkampf",
```
Comment (pki0) [inline static/locales/de.json:639]: This is not related to raids. It related to a gym is under attack by another player!

File: static/locales/de.json (hunk 5 above)
Comment (pki0) [reply static/locales/de.json:801]: See https://github.com/cecpk/RocketMAD/blob/master/templates/pokemon-history.html#L167 

Gives very bad dropdown. :/ And beginning of any option is capital. Also in German. ;) 
Where is it in a sentence?

File: static/locales/de.json (hunk 7 above)
Comment (pki0) [reply static/locales/de.json:815]: See bad dropdown

File: static/locales/de.json (hunk 10 above)
Comment (pki0) [reply static/locales/de.json:907]: See dropdown. Can you point the sentence pls?

**Review by pki0 — COMMENTED** [review]
Cannot find the sentence with the "last X hours" problem. Maybe we should convert to lower there to not get bad dropdowns.

File: static/locales/de.json (hunk 41 above)
Comment (ReuschelCGN) [reply static/locales/de.json]: corrected

File: static/locales/de.json (hunk 42 above)
Comment (ReuschelCGN) [reply static/locales/de.json:639]: okay is changed

**Comment by ReuschelCGN** [comment]
> Cannot find the sentence with the "last X hours" problem. Maybe we should convert to lower there to not get bad dropdown 
in Pokemon History is the sentence but it is divided in to parts: 
first part is :  "Pokémon seen in": "Gesehene Pokémon in",
and second part is the timezone:  "Last 12 hours": "den letzten 12 Stunden",

so you get in german:
"Gesehene Pokémon in den letzten 12 Stunden"

File: static/locales/de.json (hunk 36 above)
[the remark this answers is not in the document]
Comment (ReuschelCGN) [reply static/locales/de.json]: changed

File: static/locales/de.json (hunk 5 above)
Comment (ReuschelCGN) [reply static/locales/de.json:801]: Pokemon History -> sentence is two parts (see comments)

**Comment by pki0** [comment]
> > Cannot find the sentence with the "last X hours" problem. Maybe we should convert to lower there to not get bad dropdown
> > in Pokemon History is the sentence but it is divided in to parts:
> > first part is :  "Pokémon seen in": "Gesehene Pokémon in",
> > and second part is the timezone:  "Last 12 hours": "den letzten 12 Stunden",
> 
> so you get in german:
> "Gesehene Pokémon in den letzten 12 Stunden"

Have you tested this?
So you point to pokemon-history, correct?

In JS we have: `i18n('Pokémon seen in') + ' ' + $('#duration-select option:selected').text().toLowerCase())`It's already converted to lower case here. 
Still don't know where the sentence is, where general capital translation does mistakes.

**Comment by na-ji** [comment]
Hello,

Thank you guys for your hard work, but I just merged #207 which was already quite complete. If there are still some changes to be made on the translation, I invite you open a new PR.
